pkg/service/service.go

```diff
@@ -0,0 +1,44 @@
+package service
```

Comment:
This abstraction is used for defining a container for some services (mostly HTTP servers) that automatically controls their lifecycle (start/stop).